config/jest/resolver.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (path, options) =>
+  options.defaultResolver(path, {
+    ...options,
+    // Use packageFilter to process parsed `package.json` before the resolution (see https://www.npmjs.com/package/resolve#resolveid-opts-cb)
+    packageFilter: (pkg) => {
+      if (pkg.name === "@stellar/design-system") {
```

Comment:
interesting use-case here:

Webopack/Rollup default to using the file listed under `module` in package.json, but Jest uses `main`. I set this up because I'd like Jest to use the same file path that Webpack will be using